example/ck_tile/03_gemm/run_gemm_example.inc

```diff
@@ -28,8 +28,13 @@ float invoke_gemm(ck_tile::DeviceMem& a_m_k_dev_buf,
     args.stride_B = stride_B;
     args.stride_C = stride_C;
 
-    float ave_time = gemm_calc<ALayout, BLayout, CLayout>(
-        args, ck_tile::stream_config{nullptr, true, 1, n_warmup, n_repeat});
+    gemm_traits traits{DataTypeTraits<ADataType>{}.name,
```

Comment:
Please add a TODO note that this should be changed for mixed prec gemms. Probably would need to pass both A/B data types then.